### DISCUSSION TITLE
The flip function is not for mirror

### DISCUSSION THREAD

**Discussion by jengelbertt** [ManimCommunity/manim#3891] [Q&A] [answered]
I wanted to mirror an image, I used the flip function, but I found that flip only rotates. Is there another function for mirror?

**Comment by uwezi** [comment, marked as answer]
The `Image()` object currently has a few (reported) problems...
You could mirror the pixel matrix using e.g. the tools from the PILLOW library

-> https://github.com/ManimCommunity/manim/issues/2412
-> https://github.com/ManimCommunity/manim/issues/2979
